### PR TITLE
rakudo-star: Change description and add ZEF to bin

### PR DESCRIPTION
**Pull request by AntonOks**
taking the the description from https://rakudo.org/star and adding the **_zef_** tool to the bin / path to enable RAKU module management.

### CODE REVIEW

File: bucket/rakudo-star.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
     "bin": [
         "bin\\perl6.exe",
         "bin\\raku.exe",
-        "bin\\rakudo.exe"
+        "bin\\rakudo.exe",
+        "share\\perl6\\site\\bin\\zef",
```
Comment (rasa) [inline bucket/rakudo-star.json:17]: This line will shim to the same name as the next line. One will need to be renamed. Is this a perl script without an extenstion?

**Comment by AntonOks** [comment]
Yes, one is a PERL script and the other a bat...I can check how those scripts work without having PERL installed... if at all.Am 07.11.20 um 16:26 schrieb Ross Smith II
                
            
            
                
                    Von: "Ross Smith II" <notifications@github.com>Datum: 7. November 2020An: "ScoopInstaller/Main" <Main@noreply.github.com>Cc: "Anton Oks" <notna@ist-einmalig.de>,"Author" <author@noreply.github.com>Betreff: Re: [ScoopInstaller/Main] rakudo-star: Change description and add ZEF to bin (#1560)
                
                
@rasa commented on this pull request.



In bucket/rakudo-star.json:
> @@ -13,7 +13,9 @@
     "bin": [
         "bin\\perl6.exe",
         "bin\\raku.exe",
-        "bin\\rakudo.exe"
+        "bin\\rakudo.exe",
+        "share\\perl6\\site\\bin\\zef",

This line will shim to the same name as the next line. One will need to be renamed. Is this a perl script without an extenstion?

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ScoopInstaller/Main/pull/1560#pullrequestreview-525660368",
"url": "https://github.com/ScoopInstaller/Main/pull/1560#pullrequestreview-525660368",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

File: bucket/rakudo-star.json (hunk 1 above)
Comment (AntonOks) [reply bucket/rakudo-star.json:17]: yes, it's a PERL6 script.
I guess I'll remove the zef.bat. Would that be OK for you?

Comment (AntonOks) [reply bucket/rakudo-star.json]: Ok, tested now. If only one of those is allowed then if has to be zef.bat. Both Powershell and CMD work only with zef.bat.
CMD doesn't like the plain zef :|

**Review by chawyehsu — APPROVED** [review]

**Pull request merged**